sequencer/synchronizer/synchronizer.go

```diff
@@ -122,6 +122,7 @@ type Config struct {
 	StatsUpdateBlockNumDiffThreshold uint16
 	StatsUpdateFrequencyDivider      uint16
 	ChainID                          uint64
+	StartBlockNum                    int64
```

Comment:
@Intizar-T I think we have already GenesisBlockNum, which is first block, rather than StartBlockNum in rollupConstants [[link](https://github.com/tokamak-network/tokamak-sybil-resistance-mvp/blob/develop/sequencer/common/ethrollup.go#L100)]

Reply:
OK, thanks for letting me know.